README.md

```diff
@@ -1,40 +1,75 @@
+be ran within OKD/OCP clusters. Windows instances can be added either by creating a [MachineSet](https://docs.openshift.com/container-platform/4.5/machine_management/creating_machinesets/creating-machineset-aws.html#machine-api-overview_creating-machineset-aws),
+instructing the cluster to provision the instances, or by specifying existing instances through a ConfigMap. Through
+either method, the Windows instance must have the Docker container runtime installed. The operator will do all the
+necessary steps to configure the instance so that it can join the cluster as a worker node.
 
 More design details can be explored in the [WMCO enhancement](https://github.com/openshift/enhancements/blob/master/enhancements/windows-containers/windows-machine-config-operator.md).
 
 ## Pre-requisites
-- OKD/OCP 4.6 cluster running on Azure, AWS or vSphere configured with [hybrid OVN Kubernetes networking](docs/setup-hybrid-OVNKubernetes-cluster.md)
+- An OKD/OCP cluster of version 4.6 or newer, running on Azure, AWS or vSphere configured with
+  [hybrid OVN Kubernetes networking](docs/setup-hybrid-OVNKubernetes-cluster.md).
+  - In order to [configure BYOH Windows instances](#configuring-byoh-windows-instances) OKD/OCP 4.8+ and WMCO
+    3.0+ must be used.
 - [Install](https://sdk.operatorframework.io/docs/installation/) operator-sdk v1.6.1
 - The operator is written using operator-sdk [v1.6.1](https://github.com/operator-framework/operator-sdk/releases/tag/v1.6.1)
   and has the same [pre-requisites](https://sdk.operatorframework.io/docs/installation/#prerequisites) as it
   does.
 - [vSphere prerequisites](docs/vsphere-prerequisites.md)
 
 ## Usage
+
+### Installation
 The operator can be installed from the *community-operators* catalog on OperatorHub.
 It can also be build and installed from source manually, see the [development instructions](docs/HACKING.md).
 
+### Create a private key secret
 Once the `openshift-windows-machine-config-operator` namespace has been created, a secret must be created containing
-the private key that will be used to access the Windows VMs:
+the private key that will be used to access the Windows instances:
 ```shell script
 # Create secret containing the private key in the openshift-windows-machine-config-operator namespace
 oc create secret generic cloud-private-key --from-file=private-key.pem=/path/to/key -n openshift-windows-machine-config-operator
 ```
- We strongly recommend not using the same
+We strongly recommend not using the same
 [private key](https://docs.openshift.com/container-platform/4.6/installing/installing_azure/installing-azure-default.html#ssh-agent-using_installing-azure-default)
 used when installing the cluster
 
+### Configuring BYOH (Bring Your Own Host) Windows instances
+WARNING: This is not a fully developed feature. Nodes can be removed from the cluster by deleting the Node object,
```

Comment:
Is there a point when this will be fully developed? Is that planned for this or a future release?

Reply:
This epic tracks this feature set: https://issues.redhat.com/browse/WINC-502

It is currently planned for 3.0.0